src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -394,8 +406,8 @@ public async Task EndInvokeJSFromDotNet(long asyncCall, bool succeded, string ar
             }
             catch (Exception ex)
             {
+                // An error completing JS interop is not considered fatal.
```

Comment:
I believe this is not correct. If I recall correctly, this doesn't mean the call threw, but the input to the call was wrong. (Invalid JSON, etc.) so we do want to terminate the circuit here.

Reply:
Fixed this, it's now going to shutdown the circuit.